src/sbd.service.in

```diff
@@ -22,4 +22,5 @@ Restart=on-abort
 
 [Install]
 RequiredBy=corosync.service
-
+RequiredBy=pacemaker.service
+RequiredBy=dlm.service
```

Comment:
This is useless without also adding `Before=dlm.service`.

Reply:
guess it is symmetric to the pacemaker-case ...